src/main.rs

```diff
@@ -163,6 +163,9 @@ enum Opt {
         #[clap(name = "SHELL", parse(try_from_str))]
         shell: Shell,
     },
+    /// Zig linker wrapper
```

Comment:
Should be hidden, just like pep517

Reply:
It's done here: https://github.com/PyO3/maturin/pull/763/files#diff-49b1f98d9d763bc0e0db9c28ba3dc79ca7413959f017924bc333f7a086d16189R20

Reply:
It turns out clap 3.0 doesn't support set subcommand to hidden in here, it only can be set in subcommand struct/enum definition site, see https://github.com/clap-rs/clap/discussions/3497